integreat_compass/core/settings.py

```diff
@@ -309,3 +309,59 @@
         },
     },
 }
+
+##########
+# EMAILS #
+##########
+
+#: The backend to use for sending emails (see :setting:`django:EMAIL_BACKEND` and :doc:`django:topics/email`)
+EMAIL_BACKEND = (
+    "django.core.mail.backends.console.EmailBackend"
+    if DEBUG
+    else "django.core.mail.backends.smtp.EmailBackend"
+)
+
+#: Default email address to use for various automated correspondence from the site manager(s)
+#: (see :setting:`django:DEFAULT_FROM_EMAIL`)
+DEFAULT_FROM_EMAIL = os.environ.get(
+    "INTEGREAT_COMPASS_SERVER_EMAIL", "keineantwort@integreat-compass.de"
```

Comment:
I personally would only use @example.com mail addresses in public code. But that is just a personal preference ^^

Reply:
The `keineantwort@` is probably irrelevant if it does not have an inbox, but `tech@` could become subject to spam? However, for integreat-cms, this does not seem to be the case...